packages/lesswrong/server/callbacks.ts

```diff
@@ -65,7 +65,7 @@ getCollectionHooks("Users").updateAsync.add(function userEditDeleteContentCallba
 });
 
 getCollectionHooks("Users").editAsync.add(function userEditBannedCallbacksAsync(user: DbUser, oldUser: DbUser) {
-  if (new Date(user.banned) > new Date() && !(new Date(oldUser.banned) > new Date())) {
+  if (user.banned && oldUser.banned && new Date(user.banned) > new Date() && !(new Date(oldUser.banned) > new Date())) {
```

Comment:
This is probably wrong. I think we're trying to detect the case where `user` is banned but `oldUser` is not (`new Date(null)` is the Unix epoch and `new Date(undefined)` is "Invalid date", so `!(new Date(oldUser.banned) > new Date())` is true in either case).